### PR TITLE
Ontology georeference

### PR DESCRIPTION
**Pull request by MircoNierenz**
# Description

This ontology was created by @MircoNierenz with the latest version of the Ontology Creator and uses the current Excel file as input: Metadata, which was created in collaboration with all data providers from the GAIA-X4PLCC-AAD project and can now be merged into the main branch.

We should use the current status of the ttl-files to test it and to modify it to the needs and requierements from our ecosystem.

So everybody who has to build an domain specific Shacl- and Ontology - File can learn from here.

## Type of change

add GeoreferenceShape as separate ontology with projectLocation and geodeticReferenceSystem sub category

## Checklist

- [x] My code follows the modelling guidelines of this project
- [x] I have performed a self-review of my own changes

### CODE REVIEW

File: georeference/georeference_ontology.ttl [hunk 1]
```diff
@@ -0,0 +1,17 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix georeference: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/georeference/> .
+@prefix gx: <https://w3id.org/gaia-x/core#> .
```
Comment (robertschubert) [inline georeference/georeference_ontology.ttl:3]: gx should be 
`@prefix gx: <https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#> .`

File: georeference/georeference_ontology.ttl [hunk 2]
```diff
@@ -0,0 +1,17 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
```
Comment (robertschubert) [inline georeference/georeference_ontology.ttl:1]: georeference_instance.json is missing.

File: georeference/georeference_ontology.ttl [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+georeference: a owl:Ontology ;
+    rdfs:label "ontology definition for location and georeference information "@en ;
+    dcterms:contributor "Sebastian Tuttas (3DMS)" ;
+    owl:versionInfo "0.1" .
+
+georeference:Georeference a owl:Class ;
```
Comment (robertschubert) [inline georeference/georeference_ontology.ttl:13]: Question of taste:
GeoReference better than Georeference?
I can live with both.

File: georeference/georeference_shacl.ttl [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+            sh:message "Validation of origin failed!"@en ;
+            sh:name "origin"@en ;
+            sh:node georeference:Coordinate2DShape ;
+            sh:minCount 1 ;
+            sh:maxCount 1 ;
+            sh:order 0 ;
+            sh:path georeference:origin ],
+        [ skos:example "EPSG::32632" ;
+            sh:datatype xsd:string ;
+            sh:description "EPSG code of the map"@en ;
```
Comment (robertschubert) [inline georeference/georeference_shacl.ttl:31]: can we add a sh:in here to have a list of valid values?

File: georeference/georeference_shacl.ttl [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+            sh:minCount 0 ;
+            sh:maxCount 1 ;
+            sh:message "Validation of coordinateSystem failed!"@en ;
+            sh:name "coordinateSystem"@en ;
+            sh:order 0 ;
+            sh:path georeference:coordinateSystem ],
+        [ skos:example "EPSG xyz" ;
+            sh:datatype xsd:string ;
+            sh:description "Ellipsodial height or orthometric height"@en ;
```
Comment (robertschubert) [inline georeference/georeference_shacl.ttl:40]: can we have a sh:in here?

File: georeference/georeference_shacl.ttl [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+        [ sh:datatype xsd:float ;
+            sh:message "Validation of y failed!"@en ;
+            sh:name "y"@en ;
+            sh:maxCount 1 ;
+            sh:order 1 ;
+            sh:path georeference:y ] ;
+    sh:targetClass georeference:Coordinate2D .
+    
+georeference:ProjectLocationShape a sh:NodeShape ;
+    sh:property [ skos:example "nan" ;
```
Comment (robertschubert) [inline georeference/georeference_shacl.ttl:91]: there are several "nan" examples in this file.
Please add a real example or delete it.

File: georeference/georeference_shacl.ttl [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+            sh:maxCount 1 ;
+            sh:name "region"@en ;
+            sh:order 2 ;
+            sh:path georeference:region ],
+        [ skos:example "Germany, GER" ;
+            sh:datatype xsd:string ;
+            sh:description "Country code as ISO 3166-1, alpha-2; "@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of country failed!"@en ;
+            sh:name "country"@en ;
```
Comment (robertschubert) [inline georeference/georeference_shacl.ttl:120]: is it possible to reuse the 

`<http://www.w3.org/2006/vcard/ns#AddressShape> a sh:NodeShape ;`

from the gaia-x vocabulary? (the one defined with the gx prefix.

**Review by robertschubert — CHANGES_REQUESTED** [review]

File: georeference/georeference_ontology.ttl (hunk 1 above)
Comment (MircoNierenz) [reply georeference/georeference_ontology.ttl:3]: fixed

File: georeference/georeference_ontology.ttl (hunk 3 above)
Comment (MircoNierenz) [reply georeference/georeference_ontology.ttl:13]: i would leave it at Georeference, it's a common term

File: georeference/georeference_shacl.ttl (hunk 4 above)
Comment (MircoNierenz) [reply georeference/georeference_shacl.ttl:31]: No, there are thousands!

File: georeference/georeference_shacl.ttl (hunk 6 above)
Comment (MircoNierenz) [reply georeference/georeference_shacl.ttl:91]: fixed

File: georeference/georeference_shacl.ttl (hunk 7 above)
Comment (MircoNierenz) [reply georeference/georeference_shacl.ttl:120]: find nothing there. 
only at 
https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework
with gx:PostalAddressShape

Comment (MircoNierenz) [reply georeference/georeference_shacl.ttl:120]: would leave it as it is for now to enable the pull request

File: georeference/georeference_shacl.ttl (hunk 5 above)
Comment (MircoNierenz) [reply georeference/georeference_shacl.ttl:40]: done!

File: georeference/georeference_ontology.ttl (hunk 2 above)
Comment (MircoNierenz) [reply georeference/georeference_ontology.ttl:1]: add georeference-instance.json. done!

File: georeference/georeference_ontology.ttl (hunk 3 above)
Comment (robertschubert) [reply georeference/georeference_ontology.ttl:14]: fine with that

File: georeference/georeference_shacl.ttl (hunk 4 above)
Comment (robertschubert) [reply georeference/georeference_shacl.ttl:31]: fine with that

**Review by robertschubert — APPROVED** [review]
approved.
@MircoNierenz There are merge conflicts. Please resolve (or ask if you need help with that).

@3Dbastian 
Could you do the 2nd approval after this?

**Comment by MircoNierenz** [comment]
> approved. @MircoNierenz There are merge conflicts. Please resolve (or ask if you need help with that).
> 
> @3Dbastian Could you do the 2nd approval after this?

merge from main in branch. done!

**Pull request merged**